_inc/client/plans/single-product-search/index.jsx

```diff
@@ -86,32 +82,24 @@ export function SingleProductSearchCard( props ) {
 				</h4>
 				<div className="single-product-search__radio-buttons-container">
 					<PlanRadioButton
-						billingTimeFrame="monthly"
-						checked={ timeframe === 'monthly' }
-						currencyCode={ currencyCode }
-						fullPrice={ monthlyPrice }
-						onChange={ handleSelectedTimeframeChange }
-						planName="Monthly"
-						radioValue="monthly"
-					/>
-					<PlanRadioButton
-						billingTimeFrame="yearly"
-						checked={ timeframe === 'yearly' }
+						billingTimeFrame={ planDuration }
+						checked={ true }
 						currencyCode={ currencyCode }
-						fullPrice={ yearlyPrice }
-						onChange={ handleSelectedTimeframeChange }
-						planName="Annual"
-						radioValue="yearly"
+						fullPrice={ planDuration === 'yearly' ? yearlyPrice : monthlyPrice }
+						planName={ planDuration === 'yearly' ? 'Annual' : 'Monthly' }
```

Comment:
We'll probably want to use `Yearly` since that seems to be the norm elsewhere on the page. 
We'll also want to translate these... Whoops :)

```suggestion
						planName={ planDuration === 'yearly' ? __( 'Yearly' ) : __( 'Monthly' ) }
```

Reply:
Also updated this with your suggestion.